src/status_im2/contexts/chat/lightbox/view.cljs

```diff
@@ -102,17 +104,18 @@
          :on-viewable-items-changed         callback}]]]
      (when (and (not @transparent?) (not landscape?))
        [:f> bottom-view/bottom-view messages index scroll-index insets animations derived
-        item-width props])]))
+        item-width props state])]))
 
 (defn- f-lightbox
-  [{:keys [messages index]}]
-  (let [props (utils/init-props)
-        state (utils/init-state messages index)]
-    (fn [{:keys [messages index]}]
+  []
+  (let [{:keys [messages index]} (rf/sub [:get-screen-params])
+        props                    (utils/init-props)
+        state                    (utils/init-state messages index)
+        callback                 (fn [e]
+                                   (on-viewable-items-changed e props state))]
```

Comment:
What does this function do?
naming something as `callback` is almost the same as calling it `function`, so it'd be good to rename it with what it does in terms of components/view. e.g. navigate-to-somewhere or scroll-to-somewhere